Sources/XcbeautifyLib/Parser.swift

```diff
@@ -1,10 +1,12 @@
 public class Parser {
     
     private let colored: Bool
+
+    private let renderer: OutputRendering
     
     private let additionalLines: () -> String?
 
-    public var summary: TestSummary? = nil
+    private(set) var summary: TestSummary? = nil
```

Comment:
Since this changes a public `{ get set }` to only a public `{ get }`, is this a breaking change for API consumers? If so, what do we need to do to mark the next version of xcbeautify as a breaking change?

Otherwise, can we revert this change?